### PR TITLE
Universal ID

### PR DESCRIPTION
**Pull request by oliver-sanders**
Update the UI to use the new Universal Identifier

Sibling to:

* https://github.com/cylc/cylc-flow/pull/3931
* https://github.com/cylc/cylc-uiserver/pull/297

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Appropriate tests are included (unit and/or functional).
- [x] Appropriate change log entry included.
- [x] No documentation update required.

### CODE REVIEW

File: etc/uid.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
```
Comment (oliver-sanders) [inline etc/uid.py:1]: Python script for pulling the UID regex out of cylc-flow and dropping it in here.

Could run in a GH action if being serious, not done this yet.

Comment (MetRonnie) [reply etc/uid.py:1]: Might be useful to put this description in a docstring for this script

**Review by MetRonnie — COMMENTED** [review]
Viewed all the files

Started testing, however have run into a problem using the play button/play mutation form on the UI
> command failed: play - Error: too many values to unpack (expected 3)

Traceback indicates it might be a problem with the uiserver rather than the UI

**Comment by oliver-sanders** [comment]
Back with me for the mo whilst I fix some issues.

**Comment by oliver-sanders** [comment]
Should be sorted (will need to pull in the latest commit from the UIS branch).

**Review by MetRonnie — COMMENTED** [review]
Good, playing and viewing in the UI working now 👍

Small thing I've noticed: the placeholder for tasks in the mutations form still uses the old format

image

**Review by hjoliver — APPROVED** [review]
Looks good to me.  I'm not sure when @AaronDCole starts back yet.  Happy for this to go in now with the sibling PRs

**Comment by hjoliver** [comment]
Sibling PRs merged on the  back end, so I'll get this in too.

**Pull request merged**